FogCreek/Glitch-Community
# Debonair quokka: Teams modal pane

## Links
* https://debonair-quokka.glitch.me/
* https://app.clubhouse.io/glitch/story/3863/team-ownership-transfer-highlights-and-links-out-to-which-teams-have-multiple-members-but-only-one-admin-step-2a
* https://docs.google.com/document/d/1ya5WQraU0OPXf22uCsTGhaNnFU-3XyVLHSqGHzMkTPk/edit#heading=h.3im6lap0txtw (design spec)
https://docs.google.com/document/d/17O0aLlf0znuCN4-LuVihAkGzsUtdFpv4uV7dQpouvWI/edit#heading=h.c3jzht6x35ee (tech spec)

## GIF/Screenshots:
<img width="656" alt="Screen Shot 2019-12-05 at 3 01 01 PM" src="https://user-images.githubusercontent.com/3011231/70269758-fefd2380-1770-11ea-8f85-04eeed2a77c2.png">
<img width="659" alt="Screen Shot 2019-12-05 at 3 01 27 PM" src="https://user-images.githubusercontent.com/3011231/70269764-0290aa80-1771-11ea-9bfb-d3676a5486e3.png">

## Changes:
* Creates a team-result-item component so that teams can be shown in a ResultsList
* Looks for teams where you're the only admin and there are multiple users, and not letting you continue till they've been fixed (as deleting your account would leave these in a bad state)
* Updates list automatically when project changes are made, no need for a refresh button!

## How To Test:
* Be logged in, turn on Account Deletion and at least one other toggle
* Go to /settings, hit Delete Account, Continue to Delete Account, Continue to Delete Account (again)
* See the teams you need to take action on, try changing admins and/or deleting, make sure they disappear

## Feedback I'm looking for:
whatever you've got 😄

## Things left to do before deploying:
* not a blocker to deploying (since its behind a dev toggle) but we should have the help links go somewhere.
